staging/src/k8s.io/api/go.mod

```diff
@@ -0,0 +1,7 @@
+module k8s.io/api
+
+require k8s.io/apimachinery v1.12.0
+
+replace k8s.io/apimachinery v1.12.0 => ../apimachinery
+
+require github.com/gogo/protobuf v0.0.0-20170330071051-c0656edd0d9e
```

Comment:
For the time being we still have to publish Godeps.json and vendor/ (for the master branch) of our staging repos. Godeps.json lists all packages, not only the repos. The publishing-bot uses that as a trigger to rebuilt the pruned vendor/ dir.

So in theory we could recreate Godeps.json (for our libraries' consumers who are still on old non-vgo tools) during the publishing run. But we now lack the trigger. Running godep is awefully slow without this optimization on each commit during publishing.

Hence, I would prefer to keep Godeps.json around for the time being, even if it is generated somehow from go.mod.